meetings/2025-04-23.md

```diff
@@ -0,0 +1,111 @@
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Conduct](https://www.w3.org/policies/code-of-conduct/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+### Scribes
+* Hadrian
+* Pavlik
+
+### Introductions
+* Tom Byrd, IT professional for 40 years. Activity involved in privacy rights. Came across Solid in 2021. Working on a few ideas, want to be in sync with Solid and ODI.
+
+---
+
+## Topics
+
+### Solid Symposium 2025
+
+https://sosy2025.eu/
+
+* eP: April 24-25
+* eP: Will there be materials available online afterwards?
+* ... I believe Jeff, Rahul and Jesse have sessions.
+* RG: Solid Practitioners are organizing a session there. There are 6 talks. We try to foster collaboration between communities.
+* ... Speakers from EU and US. 
+* https://sosy2025.pages.dev/solid-together
+* eP: Good luck.
+* RG: It's tomorrow morning 10am CET.
+* ... I intend to put up the slides. We probably won't have video. We'll try to broadcast, but we won't have the ability to share the link too much in advance.
+
+### Code of Conduct related issues
+
+* eP: Tomorrow Hadrian and I will have a call with W3C staff. Michiel is on vacations.
+* eP: If you would like to bring up interactions where CoC might have been violated. You can do it publically or contact CG chairs directly. 
+* RG: Please refrain from discussing any issue that involves me until I am sent an official communication from W3C and the Solid CG.
+* HZ: Based on my experience with Apache, we talk about issues not people in public. In general people mean well just emotions get escalated. I think we will resolve it and my goal is to prevent such problems from haunting us in the fugure. We are a community and we have to communicate.
+* HZ: In general I don't want to offer to mediate, unless I'm asked. This might be seen as pressure from one of the sides. Please ask me if you want help with resolving the problem.
+* RG: I haven't received any communication except the two public comments. I don't think the issues should be discussed until the situation is resolved. I have right to privacy and ask for a right to privacy. 
```

Comment:
```suggestion
* HZ: Based on my experience with Apache, we talk about issues not people in public. In general, people mean well; emotions just get escalated. I think we will resolve it and my goal is to prevent such problems from haunting us in the future. We are a community, and we have to communicate.
* ... In general, I don't want to offer to mediate, unless I'm asked. This might be seen as pressure from one of the sides. Please ask me if you want help with resolving the problem.
* RG: I haven't received any communication except the two public comments. I don't think the issues should be discussed until the situation is resolved. I have right to privacy, and ask for a right to privacy. 
```